site/static/docs/5.3/examples/agent-card.html

```diff
@@ -0,0 +1,201 @@
+            &lt;span class="agent-type person"&gt;
+              &lt;i class="material-icons"&gt;person&lt;/i&gt; Pessoa
+            &lt;/span&gt;
+          &lt;/div&gt;
+        &lt;/div&gt;
+
+        &lt;div class="agent-card-body"&gt;
+          &lt;p class="agent-area"&gt;&lt;strong&gt;Áreas de atuação (3):&lt;/strong&gt;&lt;span class="agent-sub-area"&gt;DESIGN, ARTES VISUAIS, ARTES GRÁFICA&lt;/span&gt;&lt;/p&gt;
+          &lt;div class="location-seals-container"&gt;
+            &lt;div class="agent-seals"&gt;
+              &lt;p class="seals-area"&gt;&lt;strong&gt;Selos:&lt;/strong&gt;
+                &lt;span class="seal"&gt;Avaliador UFPR&lt;/span&gt;
+              &lt;/p&gt;
+            &lt;/div&gt;
+          &lt;/div&gt;
+          &lt;p class="agent-location"&gt;&lt;strong&gt;Localização:&lt;/strong&gt; Brasília (DF)&lt;/p&gt;
+          &lt;p class="agent-description"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua...&lt;/p&gt;
+          &lt;div class="access-profile-container"&gt;
+            &lt;button class="btn btn-primary access-profile"&gt;Acessar perfil&lt;/button&gt;
+          &lt;/div&gt;
+        &lt;/div&gt;
+      &lt;/div&gt;
+
+      &lt;div class="agent-card"&gt;
+        &lt;div class="agent-card-header"&gt;
+          &lt;img src="../assets/img/examples/agent-card3.png" alt="Foto do agente" class="rounded-circle"&gt;
+          &lt;div class="agent-info"&gt;
+            &lt;h5 class="agent-name"&gt;Estúdio de Anne Elisa&lt;/h5&gt;
+            &lt;span class="agent-type organization"&gt;
+              &lt;i class="material-icons"&gt;groups&lt;/i&gt; Organização
+            &lt;/span&gt;
+          &lt;/div&gt;
+        &lt;/div&gt;
+
+        &lt;div class="agent-card-body"&gt;
+          &lt;p class="agent-area"&gt;&lt;strong&gt;Áreas de atuação (1):&lt;/strong&gt;&lt;span class="agent-sub-area"&gt;DESENVOLVIMENTO&lt;/span&gt;&lt;/p&gt;
+          &lt;p class="agent-location"&gt;&lt;strong&gt;Localização:&lt;/strong&gt; Goiânia (GO)&lt;/p&gt;
+          &lt;div class="agent-seals"&gt;
+            &lt;p class="seals-area"&gt;&lt;strong&gt;Selos:&lt;/strong&gt;
+              &lt;span class="seal"&gt;Selo Mapas&lt;/span&gt;
+          &lt;/div&gt;
+          &lt;p class="agent-description"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua...&lt;/p&gt;
+          &lt;div class="access-profile-container"&gt;
+            &lt;button class="btn btn-primary access-profile"&gt;Acessar perfil&lt;/button&gt;
+          &lt;/div&gt;
+        &lt;/div&gt;
+      &lt;/div&gt;
+
+    </code>
+  </pre>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.23.0/prism.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/5.3.0/js/bootstrap.bundle.min.js"></script>
```

Comment:
Seria interessante incluir o script necessário para o componente dentro do `<pre><code>`, assim quem for usar irá saber que este componente precisa dessa importação.